packages/flame/lib/src/parallax.dart

```diff
@@ -156,20 +247,21 @@ class ParallaxLayer {
       ..divide(_imageSize);
 
     // Align image to correct side of the screen
-    final alignment = parallaxImage.alignment;
+    final alignment = parallaxRenderer.alignment;
     final marginX = alignment.x == 0 ? overflow.x / 2 : alignment.x;
     final marginY = alignment.y == 0 ? overflow.y / 2 : alignment.y;
-    _scroll = Vector2(marginX, marginY);
+    _scroll = Vector2(math.max(marginX, 0), math.max(marginY, 0));
```

Comment:
This line is the one that fixes the alignment for layers with fill `none`.

For some reason that I was not able to understand, the vector would get `-1` on the it's value. Which then would mess the calculation on the `update` method because it multiplies the scroll with the image size.

So an image that is aligned on top or bottom, would never be correctly placed as this needs to be zero for that alignment to happen. (I imagine the same would happen for layers aligned on either of the x axis)

I am pretty sure that I just fixed a symptom here and not the cause. I will review this line again before merging the PR but if anyone has any idea of what is happening or if I miss understood something on the code, please let me know.

Reply:
@spydon I think I got the solution, this quote on the [Alignment docs](https://api.flutter.dev/flutter/painting/Alignment-class.html) did the trick:

```
Alignment(x, y) in a rectangle with height h and width w describes the point (x * w/2 + w/2, y * h/2 + h/2) in the coordinate system of the rectangle.
```

Reply:
Cool, at first glance it looks like it makes sense!
Have you tried it out with different aligment and fill permutations?

Reply:
Not yet, but I plan on adding an example for this with Dashbook properties, so we can test more combinations in an easier way

Reply:
Super!

Reply:
Example added